SpatialGDK/Source/SpatialGDKFunctionalTests/Public/LogSpatialFunctionalTest.h

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Logging/LogMacros.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialFunctionalTest, Log, All);
```

Comment:
https://docs.unrealengine.com/en-US/ProductionPipelines/DevelopmentSetup/CodingStandard/#generalstyleissues - 
> Leave a blank line at the end of the file. All .cpp and .h files should include a blank line, to coordinate with gcc.

```suggestion
DECLARE_LOG_CATEGORY_EXTERN(LogSpatialFunctionalTest, Log, All);

```